roles/manage_efm/tasks/efm_cluster_set_params.yml

```diff
@@ -0,0 +1,119 @@
+---
+
+- name: Add encrypted EFM password to efm parameters list
+  set_fact:
+    efm_parameters: "{{ efm_parameters + efm_password_parameters }}"
+  when:
+    - efm_password_parameters is defined
+    - efm_password_parameters|length > 0
+
+- name: Set efm_listen_host
+  set_fact:
+    efm_listen_host: "{{ hostvars[inventory_hostname].private_ip }}"
+
+- name: Update efm efm.properties file
+  lineinfile:
+    path: "{{ efm_properties }}"
+    regexp: "{{ line_item.name + '=' }}"
+    line: "{{ line_item.name + '=' + line_item.value|string }}"
+    state: present
+  with_items: "{{ efm_parameters }}"
+  loop_control:
+   loop_var: line_item
+  when: efm_parameters|length > 0
+  become: yes
+  register: properties_changes
+
+- name: Update {{ pg_service }} to bind {{ efm_service }}
```

Comment:
Please use quotes for task names when using variables

Reply:
Fixed!